apps/meteor/ee/app/apps/bridges/internal.ts

```diff
@@ -16,11 +17,13 @@ export class AppInternalBridge extends InternalBridge {
 			return [];
 		}
 
-		const records = Subscriptions.findByRoomIdWhenUsernameExists(roomId, {
-			projection: {
-				'u.username': 1,
-			},
-		}).toArray();
+		const records = Future.fromPromise(
```

Comment:
Why can't we use the normal `async/await` here?

Reply:
Because it would be a breaking change as it's available to apps through here https://github.com/RocketChat/Rocket.Chat.Apps-engine/blob/alpha/src/server/rooms/Room.ts#L29